Cargo.toml

```diff
@@ -97,6 +101,8 @@ tokio1 = ["tokio1_crate", "async-trait", "futures-io", "futures-util"]
 tokio1-native-tls = ["tokio1", "native-tls", "tokio1_native_tls_crate"]
 tokio1-rustls-tls = ["tokio1", "rustls-tls", "tokio1_rustls"]
 
+dkim = ["rust-crypto", "rsa"]
```

Comment:
Could you add documentation about this feature in https://github.com/lettre/lettre/blob/master/src/lib.rs#L11 too?

Reply:
Sure, done